### ISSUE TITLE
03-how-to-package-a-python

### ISSUE THREAD

**Issue by masics** [py-pkgs/py-pkgs#135]
After running command `poetry install`, I started `python` and tried `from pycounts.pycounts import count_words`. I get error `ModuleNotFoundError: No module named 'pycounts'`. What is wrong?

**Comment by TomasBeuzen** [comment]
Hi @masics - are you working in a virtual environment made with `conda`, [as we do in the book](https://py-pkgs.org/03-how-to-package-a-python#create-a-virtual-environment)?

If so, did you `conda activate` your environment before running `poetry install` and starting a `python` session?

If you are not using `conda`, then `poetry` would have created a virtual environment for you when you ran `poetry install`. You can activate this by running `poetry shell`. Then starting a `python` session and trying `from pycounts.pycounts import count_words`.

**Comment by masics** [comment]
Yes, I did. I found that `conda activate` does not work in power shell for some reason. Do I have to use cmd instead?

**Comment by TomasBeuzen** [comment]
Ah, powershell requires some additional steps to get `conda` working, check one of these links:
- https://stackoverflow.com/a/65160772
- https://gist.github.com/martinsotir/2bd2e16332dff71e0fa5be3ed3468a6c

Following instructions in one of those, you should be able to `conda activate`

**Comment by TomasBeuzen** [comment]
For anyone else that reads this, [we suggest Windows users use the Anaconda Prompt](https://py-pkgs.org/02-setup#the-command-line-interface) (it just work out of the box), but Powershell can be used following the links above

**Comment by Psirving** [comment]
I ran into the same issue, I followed the dev container setup, then, being lazy and also thinking a VM + dev container was overkill, I did this: "If you don’t want to use a virtual environment, feel free to skip to Section 3.5.2."

No mention of `poetry shell` in the guide led to the same confusion as above.
